src/screens/user/user-list.js

```diff
@@ -14,7 +14,7 @@ export const UserListComponent = () => {
         {users.length > 0 &&
           users.map((user, index) => (
             <View key={index}>
-              <Text onPress={() => navigation.navigate('UserDetails', {user})}>
+              <Text onPress={() => navigation.navigate('Details', {user})}>
```

Comment:
Can we setup CONSTANTS for this sort of referencing here? Like we usually do with Redux action type declarations?